.github/actions/setup-source-policy/action.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Setup BuildKit Source Policy'
+description: 'Sets up BUILDKIT_SOURCE_POLICY environment variable with processed policy.json'
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Format repository name
+      id: format-repo
+      uses: ./.github/actions/format-repo
+    
+    - name: Process policy file
+      id: process-policy
+      shell: bash
+      run: |
+        # Get the formatted repo from the previous step
+        FORMATTED_REPO="${{ steps.format-repo.outputs.oci-name }}"
+        
+        # Set REPO_PREFIX for envsubst
+        export REPO_PREFIX="ghcr.io/${FORMATTED_REPO}"
```

Comment:
is `"ghcr.io/$FORMATTED_REPO}"` what `envsubst` represents? or should it say `export REPO_PREFIX=envsubst`?

Reply:
`envsubst` is a binary that reads a text file and performs environment variable replacements on it and outputs the result to stdout.